app/menus/menu.js

```diff
@@ -0,0 +1,68 @@
+const os = require('os');
+const path = require('path');
+const {app, shell, dialog} = require('electron');
+
+module.exports = class Menu {
+  constructor(commands, createWindow, updatePlugins) {
+    this.commands = commands;
+
+    this.viewMenu = require('./menus/view')(commands);
```

Comment:
any reason why these require calls are in here? 🤔 couldn't you assign them and then run the function?